README.md

```diff
@@ -654,6 +659,20 @@ Switched to a new branch 'daves'
 
 This will give you a local copy of the branch `daves`, and any update that has been pushed will also show up remotely.
 
+<a name="i-want-to-create-a-new-remote-branch-from-current-local-one"></a>
+### I want to create a new remote branch from current local one
+
+```sh
+$ git config push.default current
+$ git push -u
+```
+
+If you do not want to change the git configuration, you can also use:
+
+```sh
+$ git push -u <remote> HEAD
+```
+
```

Comment:
Hi @Rijul1999, maybe you might add an extra remark here saying that the presence of the `-u` CLI option will allow one to automatically push the current branch *to the same remote*, just by running `git push` in the same branch?

Also, I've done several tests (using git 2.11.0) regarding the `git push -u` command above (with no remote argument), and it appears that it always pushes to the remote named `origin`, even if there exist several remotes with different names, and even if there is no remote `origin` (in which case one obtains the error `fatal: No configured push destination.`)...